test/pytorch_test_base.py

```diff
@@ -329,7 +319,8 @@ def disallowed_test(self, test=test):
       raise unittest.SkipTest('skipped on XLA')
       return test(self, cls.device_type)
 
-    if test_name in disabled_torch_tests[class_name] or name in disabled_torch_tests[class_name]:
+    if test_name in disabled_torch_tests[
+        class_name] or name in disabled_torch_tests[class_name]:
```

Comment:
Lol this line is a weird one ;)

Reply:
```python
    if (
        test_name in disabled_torch_tests[class_name] or 
        name in disabled_torch_tests[class_name]
    ):
```

looks mode readable to me.

Reply:
Guys, repeat after me ... we do not manually reformat the code formatted by our auto-format tool, otherwise there is no point of having one (and we really want one) 😉

Reply:
we do not manually reformat the code formatted by our auto-format tool, otherwise there is no point of having one (and we really want one)